articles/tutorials/how-to-update-applications-client-secret.md

```diff
@@ -37,7 +37,7 @@ Or you can update a `client_secret` by creating your own request to the API:
 ```har
 {
   "method": "PATCH",
-  "url": "https://YOURACCOUNT.auth0.com/api/v2/clients/YOUR_CLIENT_ID",
+  "url": "https://${account.namespace}.auth0.com/api/v2/clients/${account.clientId}",
```

Comment:
This is one of the variables we use. If the user is logged in their Auth0 domain will be displayed, for example `auth0pnp.auth0.com` 
(actually a have an error, I need to replace `${account.namespace}.auth0.com` with `${account.namespace}`). 

If the user is not logged in the string "YOUR_AUTH0_DOMAIN" will be displayed instead. 

Is that ok?